RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
>but you'll have to convince @onovotny we shouldn't have two TFMs

I brought it up to Oren on email, when we discovered `HashAlgorithm.Create` didn't work on NS2. Up to him :-D.

>I like CryptoConfig because it allows registering and resolving algorithm instances without having to recompile intermediate libs.

Understood, but that'll need to happen *anyway* currently because Key Vault wants a JWK-style identifier:

https://github.com/onovotny/RSAKeyVaultProvider/blob/fccbf2937e55a0a60aa8f23fab4a136536e893b3/RSAKeyVaultProvider/SignatureAlgorithmTranslator.cs#L10-L17

I digress though.